components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/ConfigurationManagerImpl.java

```diff
@@ -244,14 +232,29 @@ public Resource addResource(String resourceTypeName, ResourceAdd resourceAdd)
         return resource;
     }
 
+    @Override
+    public Resource addResource(String resourceTypeName, Resource resource) throws ConfigurationManagementException {
+
+        validateResourceCreateRequest(resourceTypeName, resource);
+        String resourceId = generateUniqueID();
+        if (log.isDebugEnabled()) {
+            log.debug("Resource id generated: " + resourceId);
+        }
+        resource.setResourceId(resourceId);
+        this.getConfigurationDAO().addResource(resource);
+        if (log.isDebugEnabled()) {
+            log.debug(resource.getResourceName() + " resource created successfully.");
+        }
+        log.info("Resource: " + resource.getResourceName() + " added successfully");
```

Comment:
Do you need to add both debug and info logs here. Isn't info log enough?

Reply:
Fixed in d8a058a